packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -324,7 +332,7 @@ abstract class RawKeyEvent with Diagnosticable {
         );
         break;
       default:
-        // Raw key events are not yet implemented  on iOS or other platforms,
+        // Raw key events are not yet implemented on other platforms,
```

Comment:
Actually, iOS is the last platform that needs to have raw key events implemented.  You can change out this comment block for something like:

```dart
/// This exception would only be hit on platforms that haven't yet
/// implemented raw key events, but will only be triggered if the
/// engine for those platforms sends raw key event messages in the
/// first place.
```